feature-toggles/feature-toggles-api/build.gradle

```diff
@@ -32,6 +32,7 @@ kotlin {
 
 dependencies {
     api project(":experiments-api")
+    api project(":feature-toggles-internal-api")
```

Comment:
I think this should be implementation, else anybody depending on `feature-toggles-api` will see the internal one as transitive dependency. It's only meant to be an API between feature-toggles-api and feature-toggles-impl